modules/microshift-install-rpms-olm.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// microshift/microshift-install-rpm.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-installing-with-olm-from-rpm-package_{context}"]
+== Installing the Operator Lifecycle Manager (OLM) from an RPM package
+
+When you install {microshift-short}, the Operator Lifecycle Manager (OLM) package is not installed by default. You can install the OLM on your {microshift-short} instance using a RPM package. 
+
+.Prerequisites
+
+* You have created a {microshift-short} instance. 
+
+.Procedure 
+
+. Install the OLM package by running the following command: 
++
+[source,terminal]
+----
+$ sudo dnf install microshift-olm
```

Comment:
Is this command meant to be executed on the MicroShift instance ?

Reply:
I believe so yes, bc its executed after you already have MicroShift installed

Reply:
Thanks. Please resolve comment.